bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/service/BusinessPartnerService.kt

```diff
@@ -209,6 +222,24 @@ class BusinessPartnerService(
         }
 
 
+    private fun requestNewCleaning(orchestratorBusinessPartnersDto: List<BusinessPartnerGenericDto>): TaskCreateResponse {
```

Comment:
Maybe don't call it *cleaning* to be consistent. In the Orchestrator we removed this naming everywhere.
Suggestion: `createGoldenRecordTasks`